src/pages/private/dashboard.js

```diff
@@ -43,21 +43,25 @@ const Dashboard = (props) => {
       const data = await response.json();
       // Sets the imagesArray to the array of images we received
       setImagesArray(data.hits);
-
       // Data finished loading, we set this to false to stop showing loader
       setDataLoading(false);
-
-      // Sets dataLoaded to true so we can remove the message that shows when there's no data
-      setDataLoaded(true);
     } catch (err) {
       // Sets error state to true and shows error on screen
       setError(true);
     }
   };
+
+  // Helps to download a file
+  const downloadHandler = (url) => {
+    FileSaver.saveAs(url, url.slice(-15));
```

Comment:
slice(-15) doesn't seem to be a generic solution. What if the url is: `https://httpbin.org/image/photo.png`

Reply:
Resolved with split method